pkg/controller/master/backup/backup.go

```diff
@@ -116,6 +119,7 @@ func RegisterBackup(ctx context.Context, management *config.Management, _ config
 	vmBackups.OnRemove(ctx, backupControllerName, vmBackupController.OnBackupRemove)
 	snapshots.OnChange(ctx, snapshotControllerName, vmBackupController.updateVolumeSnapshotChanged)
 	lhbackups.OnChange(ctx, longhornBackupControllerName, vmBackupController.OnLHBackupChanged)
+	lhbackups.OnRemove(ctx, cleanupSnapshotControllerName, vmBackupController.OnLHBackupRemoved)
```

Comment:
Why use a different controller name here (cleanupSnapshotControllerName), instead of using the existing longhornBackupControllerName?